verilog/CST/verilog_tree_print.h

```diff
@@ -41,6 +44,29 @@ class VerilogPrettyPrinter : public verible::PrettyPrinter {
 void PrettyPrintVerilogTree(const verible::Symbol& root, absl::string_view base,
                             std::ostream* stream);
 
+class VerilogTreeToJsonConverter : public verible::SymbolVisitor {
+ public:
+  explicit VerilogTreeToJsonConverter(absl::string_view base);
+
+  void Visit(const verible::SyntaxTreeLeaf&) override;
+  void Visit(const verible::SyntaxTreeNode&) override;
+
+  Json::Value get_json() { return json_; }
```

Comment:
Is the `json_` a potentially large object that could benefit from `std::move` to avoid a deep copy?

Reply:
It could be. Done.